Description: Enable /etc/ZionOne/conf.php so that we use packaged libraries
 Update some PHP include files to refer to /etc/ZionOne/conf.php
 instead of the in-tree conf/conf.php.
 .
 /etc/ZionOne/conf.php configures ZionOne to use packaged libraries
 instead of the embedded ones.
Author: Laurent Destailleur <eldy@users.sourceforge.net>
Forwarded: not-needed
Last-Update: 2013-07-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/htdocs/filefunc.inc.php
+++ b/htdocs/filefunc.inc.php
@@ -63,8 +63,8 @@ $conffiletoshowshort = "conf.php";
 $conffile = "conf/conf.php";
 $conffiletoshow = "htdocs/conf/conf.php";
 // For debian/redhat like systems
-//$conffile = "/etc/ZionOne/conf.php";
-//$conffiletoshow = "/etc/ZionOne/conf.php";
+$conffile = "/etc/ZionOne/conf.php";
+$conffiletoshow = "/etc/ZionOne/conf.php";


 // Include configuration
--- a/htdocs/install/inc.php
+++ b/htdocs/install/inc.php
@@ -73,8 +73,8 @@ $conffiletoshowshort = "conf.php";
 $conffile = "../conf/conf.php";
 $conffiletoshow = "htdocs/conf/conf.php";
 // For debian/redhat like systems
-//$conffile = "/etc/ZionOne/conf.php";
-//$conffiletoshow = "/etc/ZionOne/conf.php";
+$conffile = "/etc/ZionOne/conf.php";
+$conffiletoshow = "/etc/ZionOne/conf.php";

 $short_options = "c:h";
 $long_options = array(
